kotlin/workflow-runtime/src/main/java/com/squareup/workflow/internal/SubtreeManager.kt

```diff
@@ -44,39 +104,68 @@ internal class SubtreeManager<StateT, OutputT : Any>(
    */
   private val snapshotCache = mutableMapOf<AnyId, ByteString>()
 
-  private val nodeLifetimeTracker =
-    LifetimeTracker<WorkflowOutputCase<*, *, StateT, OutputT>, AnyId, WorkflowNode<*, *, *, *>>(
-        getKey = { it.id },
-        start = { it.createNode() },
-        dispose = { case, node ->
-          node.cancel()
-          snapshotCache -= case.id
-        }
-    )
+  /**
+   * When not in the middle of a render pass, this list represents the active child workflows.
+   * When in the middle of a render pass, this represents the list of children that may either
+   * be re-rendered, or destroyed after the render pass is finished if they weren't re-rendered.
+   *
+   * During rendering, when a child is rendered, if it exists in this list it is removed from here
+   * and added to [stagingChildren].
+   */
+  private var childrenFromLastCompleteRender = InlineLinkedList<WorkflowChildNode<*, *, *, *>>()
```

Comment:
After reading your doc above, I'd actually find `activeChildren` easier to understand.

Reply:
Done.